### ISSUE TITLE
How can I implement this in my project?

### ISSUE THREAD

**Issue by 0xPratikPatil** [Railly/tinte#33]
I don't want to generate themes and use them in my project. I want to create a theme customizer component that allows users to change the theme from a list, rather than generating new ones, so I can implement this in my project. @Railly

**Comment by Railly** [comment]
That sounds great, I imagine a copy-pastable component that allows you to tweak your shadcn/ui styles in dev-mode only? Similar to a debugger?

**Comment by 0xPratikPatil** [comment]
Yes we can save themes in global css or db there will be simple theme customizer components like shadcn example.can you tell me how to build this again because I didn't understand those components in your repo as there are so many components.Just tell me how to use it. @Railly

**Comment by 0xPratikPatil** [comment]
You build a great tool where we can test our theme but my concern is rather than changing the css why not make own customizer so they only add css.I am talking about Theme Customizing Mechanism.
